components/selectionBlock/selectionBlock.css

```diff
@@ -65,6 +65,7 @@
   display: flex;
   justify-content: center;
   margin: 0;
+  min-width: 290px;
```

Comment:
why is this required?

Reply:
Ah, forgot to mention in description, small fix for selection block.